dxr/vcs.py

```diff
@@ -45,11 +45,11 @@ def get_vcs_name(self):
         return type(self).__name__
 
     @classmethod
-    def invoke_vcs(cls, args, cwd):
+    def invoke_vcs(cls, args, **kwargs):
         """Return the result of invoking the VCS command on the repository,
-        with the current working directory set to the root directory.
+        with extra kwargs passed along to the Popen constructor.
         """
-        return subprocess.check_output([cls.command] + args, cwd=cwd)
+        return subprocess.check_output([cls.command] + args, **kwargs)
```

Comment:
I'd prefer we explicitly have a `cwd` arg rather than passing all kwargs through. This makes it easier to learn, since you don't have to traverse multiple levels of documentation. Also, it lets us change this method later (to, say, use something other than check_output()) without first checking all call sites to make sure they're not passing some unanticipated kwargs in.

This is all assuming we want to pass relatively few kwargs through to check_output() for the forseeable future.